symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkConfig.java

```diff
@@ -12,6 +12,7 @@
     private BdkClientConfig agent = new BdkClientConfig();
     private BdkClientConfig pod = new BdkClientConfig();
     private BdkClientConfig keyManager = new BdkClientConfig();
+    private BdkClientConfig sessionAuth = new BdkClientConfig();
```

Comment:
makes me wonder if as a user of the SDK I really need to understand all the complexity of agent/pod/keyManager/sessionAuth and need to configure all of that

Reply:
In most of the cases (unfortunately) yes you'll have to configure all of these parameters. We have to keep in mind that most of the customers that are implementing their workflows using bots and apps are based on on-prem architecture along with cert authentication. I don't see how we could avoid configuring those components =/ 

On the other hand, the `yeoman` generator really helps to quickly setup a BDK project with all of these parameters already configured (mostly used during hackathons) when Agent/KM are on-cloud and bots are using RSA authentication.

Reply:
this conversation reminds me a message from Pat O'Brien in SDK channel:

> When using rsa auth I want the sdk to ignore sessionauth url and use pod url. sessionauth urtl is for ssl cert auth and it confuses customers